src/components/templates/Home.svelte

```diff
@@ -80,6 +81,6 @@
 
   <section>
     <NewReportCounter />
-    <NewReportCounter />
+    <ComparedTo />
   </section>
 </section>
```

Comment:
Did you already change that outer section in the home component into a div? Otherwise that would be a small point that could be improved!